bin/fastq_dir_to_samplesheet.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import glob
+import argparse
+
+def parse_args(args=None):
+    Description = "Generate nf-core/rnaseq samplesheet from a directory of FastQ files."
+    Epilog = "Example usage: python fastq_dir_to_samplesheet.py <FASTQ_DIR> <SAMPLESHEET_FILE>"
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument("FASTQ_DIR", help="Folder containing raw FastQ files.")
+    parser.add_argument("SAMPLESHEET_FILE", help="Output samplesheet file.")
+    parser.add_argument('-st', '--strandedness', type=str, dest="STRANDEDNESS", default='unstranded', help="Value for 'strandedness' in samplesheet. Must be one of 'unstranded', 'forward', 'reverse'.")
+    parser.add_argument('-r1', '--read1_extension', type=str, dest="READ1_EXTENSION", default='_R1_001.fastq.gz', help="File extension for read 1.")
```

Comment:
How about specifying a glob pattern instead of an extension? I don't see any downsides and it would be more flexible.

Reply:
Can't visualise how this would look in the code. Fancy coming up with a small PR adding this after this has been merged?